webapp/src/components/CollectionGrid.jsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import CollectionCard from "./CollectionCard";
+import CollectionCardWide from "./CollectionCardWide";
+
+export default function CollectionGrid({ title, data, isFavourite }) {
+  const getNoOfCollections = () => {
+    return "4";
+  };
+  const getCollections = () => {};
+
+  return (
+    <div className="my-5">
+      <div className="row position-relative">
+        <h3 style={{ color: "#30387DB0" }}>{title}</h3>
+        {/* horizontal line */}
+        <div
+          className="position-absolute"
+          style={{
```

Comment:
Convert inline styling to class-based. Check and make changes for all the usages within this file.